Changelog.md

```diff
@@ -1,5 +1,23 @@
 Changelog
 
+v1.2.0
+
+## Frontend Changes
+
+- Display error message in the results if the record has open cases or missing/unrecognized dispositions (#685, #686)
+- Show the arrest date for both dismissals and convictions (#683)
+- Show the time analysis even if there is an error due to open cases (open cases are ignored in the analysis) (#649)
+- Show the "type" of charge as it is understood for the purposes expungement logic (#698, pending)
+- Minor display format fixes (#669)
+
+## Expungement Logic Fixes
+
+- Correctly identify violation charges (#668)
```

Comment:
Why not just use the PR title here "More robust string comparison to identify violations in expunger"?

Reply:
I figured that phrase doesn't mean much to a non-coder.

Reply:
What about "Make violation charge identification more robust"